dev/packages/alpha/cisco-0.0.2/dataset/ftd/manifest.yml

```diff
@@ -0,0 +1,52 @@
+title: Cisco ftd logs
+release: ga
+type: logs
+streams:
+  - input: logs
+    vars:
+      - name: paths
+        type: text
+        title: Paths
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - /var/log/cisco-ftd.log
+      - name: tags
+        type: text
+        title: Tags
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - cisco-ftd
+      - name: syslog_port
+        type: integer
+        title: Syslog Port
+        multi: false
+        required: true
+        show_user: true
+        default: 9003
+      - name: log_level
+        type: integer
+        title: Log Level
+        multi: false
+        required: true
+        show_user: true
+        default: 7
+      - name: internal_prefix
+        type: text
+        title: Internal Prefix
+        multi: false
+        required: true
+        show_user: true
+        default: ftd
+      - name: internal_PREFIX
```

Comment:
This reminds me of the other variable used. I think I need more background on how this should be used?